src/runtime/can_use_target.cpp

```diff
@@ -40,21 +40,26 @@ WEAK int halide_default_can_use_target_features(int count, const uint64_t *featu
 
         static_assert(sizeof(halide_cpu_features_storage) == sizeof(CpuFeatures), "CpuFeatures Mismatch");
         if (!halide_cpu_features_initialized) {
-            CpuFeatures tmp = halide_get_cpu_features();
+            CpuFeatures tmp;
+            int error = halide_get_cpu_features(&tmp);
+            if (error != halide_error_code_success) {
+                halide_error(nullptr, "Internal error: halide_get_cpu_features failed!\n");
```

Comment:
I think this should actually be halide_abort_if_false(nullptr, error == halide_error_code_success). A halide_error call might just log and return and is supposed to be followed by returning a non-zero exit status, but we can't do that for this function because it has no way to report an error status, so halide_get_cpu_features isn't allowed to return anything other than success.

Reply:
Ahh, good idea.  Okay, I'll change this to abort on failure.